source/_posts/2022-05-16-home-assistant-os-release-8.markdown

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: "Home Assistant OS Release 8"
+description: ""
```

Comment:
Missing description (its used by socials)

Reply:
Hm, so maybe something like this?

```suggestion
description: "Release notes for the Home Assistant OS release 8"
```

Reply:
Yeah, that works or, alternatively, make a small sum up of the cases in that release/highlights (social media shows this as a secondary line under the title, the title already makes clear this is the OS 8 release).

Maybe something like this:

```suggestion
description: "Now using GRUB2 for UEFI-based systems, Home Assistant Yellow support, and now working on generic AArch64 systems!"
```

Reply:
Also good 👍